src/compressed_tensors/compressors/base.py

```diff
@@ -86,58 +79,22 @@ def compression_param_info(
     def compress(
         self,
         model_state: Dict[str, Tensor],
-        names_to_scheme: Dict[str, QuantizationArgs],
         **kwargs,
     ) -> Dict[str, Tensor]:
```

Comment:
Should these not implemented methods be made abstractmethods to require them to be implemented?

Reply:
We can make compress, decompress abstract, I did that in my latest commit;

I will let compress_weights/decompress_weights be as is (Not abstract) as it will require changing Marlin24Compressor; when we get to updating that, I will make this change then!